src/AutorestSwift/Models/Simple/Language.swift

```diff
@@ -46,16 +46,19 @@ class Language: Codable {
 
     struct PagingNames: Codable {
         /// name for the item property
-        var itemName: String
+        var itemName: String? = "value"
```

Comment:
Why we use 'value' as the default value for itemName?

Reply:
And if itemName has a default value, will it ever be 'nil'? Why we need to switch from `String` to `String?`?

Reply:
https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable

The description calls out that the default is "value". Storage was not specifying this, so it was crashing. It has to be optional otherwise when we try to decode the YAML to Swift, it will crash because the "itemName" key isn't found.